intake_stac/catalog.py

```diff
@@ -430,7 +430,9 @@ def _get_driver(self, asset):
 
         if entry_type in ['', 'null', None]:
 
-            suffix = os.path.splitext(asset.media_type)[-1]
+            suffix = '.tif'
```

Comment:
I think the previous code was buggy. The media type is optional (https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#asset-object), so we shouldn't assume it's a string.